examples/projections/nongeo/cartesian_power.py

```diff
@@ -0,0 +1,34 @@
+"""
+Cartesian power
+===============
+
+``xwidth[p *p-value*]/[height [p *p-value*]]``: Give the ``width`` of the figure and
+the optional argument ``height``. The axis or axes with a logarithmic transformation
+requires ``"p"`` and the power transformation for that axis.
+"""
```

Comment:
![image](https://user-images.githubusercontent.com/3974108/102506598-46d7ab80-4051-11eb-92e2-ea6f00dbbb72.png)

This part is difficult to read. We need to find a better way to show the syntax.

Reply:
Related to this issue: https://github.com/GenericMappingTools/pygmt/issues/631

Reply:
Not sure of a  good way to do this; I'll have to give it some thought.

Reply:
@seisman Would you take a look at the updated docstring? I think it's in line with the suggestion on #631.

Reply:
Let's keep this PR open until we reach an agreement in #631.

Reply:
Sounds good!